ops/terraform/services/static-site/data-sources.tf

```
@@ -0,0 +1,227 @@
+  }
+}
+
+data "aws_vpc" "this" {
+  filter {
+    name   = "tag:Name"
+    values = [local.vpc_name]
+  }
+}
+
+data "aws_ssm_parameter" "zone_name" {
+  name            = "/bfd/mgmt/common/sensitive/r53_hosted_zone_root_domain"
+  with_decryption = true
+}
+
+data "aws_ssm_parameter" "zone_is_private" {
+  name            = "/bfd/mgmt/common/sensitive/r53_hosted_zone_root_is_private"
+  with_decryption = true
+}
+
+# data "aws_route53_zone" "this" {
+#   name         = nonsensitive(data.aws_ssm_parameter.zone_name.value)
+#   private_zone = nonsensitive(data.aws_ssm_parameter.zone_is_private.value)
+# }
+
+data "aws_iam_policy_document" "static_kms_key_policy" {
+  depends_on = [aws_cloudfront_distribution.static_site_distribution, aws_cloudfront_origin_access_identity.static_site_identity]
+  statement {
+    actions = ["kms:*"]
+    effect  = "Allow"
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"]
+    }
+    resources = ["*"]
+  }
+
+  statement {
+    actions = [
+      "kms:Decrypt",
+      "kms:GenerateDataKey",
+      "kms:ReEncrypt*"
+    ]
+    effect = "Allow"
+    principals {
+      type        = "AWS"
+      identifiers = [aws_cloudfront_origin_access_identity.static_site_identity.iam_arn]
+    }
+    resources = ["*"]
+    condition {
+      test     = "StringEquals"
+      variable = "AWS:SourceArn"
+      values   = [aws_cloudfront_distribution.static_site_distribution.arn]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "cloudfront_policy" {
+  depends_on = [aws_cloudfront_origin_access_identity.static_site_identity, aws_s3_bucket.static_site]
+  statement {
+
+    sid    = "AllowCloudFrontServicePrincipal"
+    effect = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["cloudfront.amazonaws.com"]
+    }
+
+    principals {
+      type        = "AWS"
+      identifiers = [aws_cloudfront_origin_access_identity.static_site_identity.iam_arn]
+    }
+
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::cloudfront:user/CloudFront Origin Access Identity ${aws_cloudfront_origin_access_identity.static_site_identity.id}"]
+    }
+
+    actions = ["s3:GetObject"] ##, "s3:ListBucket"]
+    resources = [
+      "${aws_s3_bucket.static_site.arn}/*"
+    ]
+  }
+
+  statement {
+    sid    = "JenkinsRWObjects"
+    effect = "Allow"
+    principals {
+      type        = "AWS"
+      identifiers = ["*"]
+    }
+    actions = ["s3:*"]
+    resources = [
+      "${aws_s3_bucket.static_site.arn}",
+      "${aws_s3_bucket.static_site.arn}/*"
+    ]
+    condition {
+      test     = "ArnEquals"
+      variable = "AWS:UserId"
+      values   = ["arn:aws:iam::${local.account_id}:role/cloudbees-jenkins"]
+    }
+  }
+
+  statement {
+    sid = "AllowKMSAccess"
+    actions = [
+      "s3:GetObject",
+      "s3:ListBucket"
+    ]
+    resources = [
+      "${aws_s3_bucket.static_site.arn}",
```

I'm puzzled that `terraform fmt` doesn't address these so-called "Interpolation-only expressions" because they've been inauspicious since terraform 0.12...
```suggestion
      aws_s3_bucket.static_site.arn,
```